radix-engine/src/engine/interpreters/wasm_runtime.rs

```diff
@@ -20,23 +17,23 @@ where
     Y: SystemApi + EngineApi<RuntimeError> + Invokable<ScryptoInvocation, RuntimeError>,
 {
     api: &'y mut Y,
+    buffers: BTreeMap<BufferId, Vec<u8>>,
```

Comment:
What are your thoughts on this being managed by the runtime vs. kernel? I have been moving towards the direction where the Runtime/Intepreter were as stateless as possible.

Reply:
So far, it's only used and needed for WASM shim (between kernel/interpreter and wasm module). It seems wrong to move the abstraction to kernel.

In terms stateless, `WasmRuntime` is alredy stateful (the behavior depends on the current state `api`). I don't see a harm for components to maintain well-defined internal state.